roadmap-2019.md

```diff
@@ -76,3 +70,17 @@ Get Pipelines into a reliable release cadence, with trustworthy integration
 tests and performance metrics to guard against regressions. The Pipelines team
 itself should dogfood Pipelines for our own CI and CD infrastructure, alongside
 or instead of Prow, by having Prow use Pipelines somehow.
+
+## Migrate to Pipelines
+
+The Knative Build Working Group has historically focused on `Build` and
+associated resources. With the successful experimentation around `Pipelines` and
+`Tasks` and `Resources` at the end of 2018, we believe it's time to further
+invest in these resources and migrate existing `Build` users to these new more
+flexible and powerful resources.
+
+## Don't Break Serving
```

Comment:
We can remove this too actually, since Serving doesn't currently have a dependency on Pipeline. If it does in the future, we should definitely add it, but I think it could also go into a "reliability" objective along the lines of a 1.0 release.

Reply:
kk!